Bottom: d2fe80bf2d778c8fa86232f744d7b3fb2d895e58
Top:    cfb662c7967bac088e8d7940ad442f8b4d7afa95
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 02:37:25 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
